amarachukwu-agbo/random-number-generator
# #165145197 Create view to display generated phone numbers and connect to API endpoint

#### What does this PR do?
Creates the view for generating phone numbers and connects it to the endpoint

#### Description of Task to be completed?
- Create components
- Write tests
- Create service that connects to endpoint

#### How should this be manually tested?
- Clone the repo and install application dependencies. 
- Run command  `yarn run client:start-dev`. 
- On another terminal, run command  `yarn run server:start-dev`. 
- Navigate to `http://localhost:1234/` and click on `Generate`. Numbers should be generated and saved to file system

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#165145197](https://www.pivotaltracker.com/story/show/165145197)

#### Screenshots (if appropriate)
N/A

#### Questions: 
N/A
